File: src/com/anathema_roguelike/entities/characters/foes/traits/Trait.java
package com.anathema_roguelike.entities.characters.foes.traits;

import com.anathema_roguelike.entities.characters.perks.PassthroughPerk;

public abstract class Trait<T extends Perk> extends PassthroughPerk<T> {

	public Trait() {
		super();
	}
}

File: src/com/anathema_roguelike/entities/characters/foes/traits/Trait.scala
package com.anathema_roguelike
package entities.characters.foes.traits

import com.anathema_roguelike.entities.characters.perks.PassthroughPerk
import com.anathema_roguelike.entities.characters.perks.Perk

abstract class Trait[T <: Perk]() extends PassthroughPerk[T] {
}